tests/test_product.py

```diff
@@ -369,3 +369,47 @@ def test_variant_availability_status(unavailable_product):
     stock.save()
     status = get_variant_availability_status(variant)
     assert status == VariantAvailabilityStatus.AVAILABLE
+
+
+def test_product_filter_empty_get(authorized_client, product_in_stock,
```

Comment:
Please name test accordingly to tested functionality.

Reply:
Renamed to test_product_filter_before_filtering